awx/playbooks/roles/sync_project/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: See if the asset files exist
+  stat:
+    path: "{{ item }}"
+  register: doesAssetFileExist
+  loop: "{{ asset_files }}"
+  loop_control:
+    label: "{{ item | regex_replace('^.*/assets', 'assets') }}"
+
+- name: Load vars from existing files
+  include_vars:
+    file: "{{ item.item }}"
+  when: item.stat.exists
+  loop: "{{ doesAssetFileExist.results }}"
+  loop_control:
+    label: "{{ item.item | regex_replace('^.*/assets', 'assets') }}"
+
+- name: Import job templates
+  awx.awx.tower_job_template:
+    allow_simultaneous: "{{ item.allow_simultaneous | default(omit) }}"
+    ask_credential_on_launch: "{{ item.ask_credential_on_launch | default(omit) }}"
+    ask_diff_mode_on_launch: "{{ item.ask_diff_mode_on_launch | default(omit) }}"
+    ask_inventory_on_launch: "{{ item.ask_inventory_on_launch | default(omit) }}"
+    ask_job_type_on_launch: "{{ item.ask_job_type_on_launch | default(omit) }}"
+    ask_limit_on_launch: "{{ item.ask_limit_on_launch | default(omit) }}"
+    ask_scm_branch_on_launch: "{{ item.ask_scm_branch_on_launch | default(omit) }}"
+    ask_skip_tags_on_launch: "{{ item.ask_skip_tags_on_launch | default(omit) }}"
+    ask_tags_on_launch: "{{ item.ask_tags_on_launch | default(omit) }}"
+    ask_variables_on_launch: "{{ item.ask_variables_on_launch | default(omit) }}"
+    ask_verbosity_on_launch: "{{ item.ask_verbosity_on_launch | default(omit) }}"
+    become_enabled: "{{ item.become_enabled | default(omit) }}"
+    credential: "{{ item.credential | default(omit) }}"
```

Comment:
If we're making something new, let's stop using this old singular `credential` key.

Reply:
Good point, this was a bulk field copy from an `ansible-doc | grep ... | grep ...` command. For now I will leave this as is since this might be changing anyway to match the import/export code.